src/runtime/gecko/l20n-html.js

```diff
@@ -0,0 +1,56 @@
+import { prioritizeLocales } from '../../intl/locale';
+import { contexts, Localization } from '../../bindings/html';
+import { documentReady, getResourceLinks, getMeta } from './util';
+import { keysFromContext, valueFromContext } from '../../lib/format';
+
+Components.utils.import('resource://gre/modules/Services.jsm');
+Components.utils.import('resource://gre/modules/L20n.jsm');
+Components.utils.import('resource://gre/modules/IntlMessageContext.jsm');
+
+const functions = {
```

Comment:
if we manage to merge `l20n-xul` and `l20n-html` into a single file then it's OK for this to be here